frontend/src/utils/api/index.js

```diff
@@ -1,7 +1,7 @@
 import APIError from './errors'
-// TODO eventually second URL should be set to deploy url
-const API_URL = process && process.env
-  ? 'http://localhost:3000/api' : 'http://127.0.0.1:3000/api'
+// TODO - THIS DOESN'T SEEM TO BE WORKING...
+export const API_URL = process && process.env && process.env.NODE_ENV === 'development'
```

Comment:
is there a reason this is now export?

Reply:
Nope

Reply:
Originally it was because of the AppSearch component having the hardcoded URL